src/ride/ride.c

```diff
@@ -7551,7 +7549,14 @@ void game_command_set_ride_vehicles(int *eax, int *ebx, int *ecx, int *edx, int
 		invalidate_test_results(rideIndex);
 		rideEntry = get_ride_entry(ride->subtype);
 		ride->subtype = value;
+
+		if (!(*ebx & GAME_COMMAND_FLAG_NETWORKED)) {
```

Comment:
The `GAME_COMMAND_FLAG_APPLY` has already been checked before this point and the value of `eax` is not read before this point.